antora-ui-camel/src/css/footer.css

```diff
@@ -71,7 +71,7 @@ footer .footer dl dd {
   margin-left: 80%;
 }
 
-@media screen and (max-width: 640px) {
+@media screen and (max-width: 1181px) {
```

Comment:
![Screenshot from 2020-03-28 09-46-10](https://user-images.githubusercontent.com/51945896/77814926-ad8bf200-70db-11ea-95b6-cfd11b011efc.png)

this I had to because, as can be seen, in the picture above, the blocks were stashing above each other. Now with this they don't and this particular thing applies for <div class="edit" only so it does not affect anything else.

Reply:
There needs to be a better way, adding media breakpoints is not a good solution. With any change we do, say we increase/decrease the font size we need to test at every breakpoint, the total number of tests is layouts*breakpoints, so it makes it very difficult for us to test, that's why we can't be adding new breakpoints.

Reply:
So, Should I work in the previous breakpoint?